QuoridorEngine/src/core/QuoridorGame.cs

```diff
@@ -140,6 +145,21 @@ public void ForcePlayerMovement(int row, int column, bool isWhite)
             targetPlayer.Column = column;
         }
 
+        /// <summary>
+        /// The board is reset and cleared, the pawns are set to their starting positions,
+        /// the number of walls is reset to default value and the move history is cleared
+        /// </summary>
+        public void ClearBoard()
```

Comment:
Choose a more appropriate method name according to the complete set of performed actions. 
_Suggestion_: **ResetGame()**